pkg/authorization/api/register.go

```diff
@@ -56,30 +56,3 @@ func addKnownTypes(scheme *runtime.Scheme) {
 		&ClusterRoleList{},
 	)
 }
-
-func (obj *ClusterRoleList) GetObjectKind() unversioned.ObjectKind          { return &obj.TypeMeta }
-func (obj *ClusterRoleBindingList) GetObjectKind() unversioned.ObjectKind   { return &obj.TypeMeta }
-func (obj *ClusterPolicyBindingList) GetObjectKind() unversioned.ObjectKind { return &obj.TypeMeta }
-func (obj *ClusterPolicyList) GetObjectKind() unversioned.ObjectKind        { return &obj.TypeMeta }
-func (obj *ClusterPolicyBinding) GetObjectKind() unversioned.ObjectKind     { return &obj.TypeMeta }
-func (obj *ClusterPolicy) GetObjectKind() unversioned.ObjectKind            { return &obj.TypeMeta }
```

Comment:
Yes, there should be a commit for that (once the upstream merges on fixing the last remaining ugliness and reflection call).